axiom/nr_app_harvest.c

```diff
@@ -12,6 +12,25 @@
 #include "nr_app_harvest_private.h"
 #include "util_logging.h"
 
+/*
+ * The build pipeline for PHP 8.2 uses a Debian image that is built on a
+ * version of glibc that contains an newer glibc version symbol for `pow()`
+ * which is incompatible with older distributions, causing RPM installs to fail.
+ *
+ * To pin the version of `pow` to a version compatible with all NR PHP Agent
+ * supported OSes, we utilize `asm()` and `.symver` to instruct the linker to
+ * select an older version of the symbol.
+ */
+#if defined(__GLIBC__)
+#if ((__GLIBC__) >= 2) && ((__GLIBC_MINOR__) > 17)
+#if defined (__x86_64__)
+  __asm__(".symver pow,pow@GLIBC_2.2.5");
+#elif defined (__aarch64__)
+  __asm__(".symver pow,pow@GLIBC_2.17");
+#endif
```

Comment:
```suggestion
```

Reply:
I think this is only needed if we are trying to package the ARM64 binaries.  Probably better not to symbol version when we don't have to.

Reply:
Resolved in de1769ac677c613c9bc34cc229e668f207a86780.